client/src/components/Sidebar/atoms.ts

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components/native';
+import SafeAreaView from 'react-native-safe-area-view';
+import {Button} from '../../components/Button';
+import {theme} from '../../consts';
+
+export const Container = styled(SafeAreaView).attrs({
+    forceInset: {
+        top: 'always',
+        bottom: 'never',
+        horizontal: 'always',
+    },
+})`
+    flex: 1;
+`;
+
+export const Header = styled.View`
+    flex-direction: row;
+    padding: ${theme.screenIndent};
+    align-items: flex-end;
+`;
+
+export const Logo = styled.Image.attrs({
```

Comment:
Wouldn't we use it somewhere else? Might be it makes sense to extract it to common components 🤔

Reply:
no